xml/html_title.xml

```diff
@@ -331,4 +331,18 @@
     <param pos="0" name="os.product" value="Linux"/>
     <param pos="0" name="os.cpe23" value="cpe:/o:debian:debian_linux:-"/>
   </fingerprint>
+  <fingerprint pattern="^Eagle Eye Network">
+    <description>EagleEyeNetworks to manage IoT cameras home webpage</description>
+    <example>Eagle Eye Networks (Local)</example>
+    <param pos="0" name="hw.product" value="Cloud Video Surveillance System"/>
+    <param pos="0" name="hw.vendor" value="Eagle Eye Networks"/>
+    <param pos="0" name="hw.device" value="Eagle Eye Bridge"/>
+  </fingerprint>
+  <fingerprint pattern="^Web Local Display Unauthorized$">
+    <description>EagleEyeNetworks to manage IoT cameras unauthorized webpage</description>
+    <example>Web Local Display Unauthorized</example>
+    <param pos="0" name="hw.product" value="Cloud Video Surveillance System"/>
+    <param pos="0" name="hw.vendor" value="Eagle Eye Networks"/>
+    <param pos="0" name="hw.device" value="Eagle Eye Bridge"/>
```

Comment:
`.device` is meant to describe the general type of device that this is in a non-vendor specific way.  so, printer, camera, laptop, etc, are all good `.device` types.  Perhaps this should be the `.product`?

Reply:
https://github.com/rapid7/recog/pull/235/commits/99a309688194ae1e21b46efe37ee1c0cfbbe2659